### PR TITLE
Renamed Dialogue controls texts in the database

### PR DESCRIPTION
**Pull request by N-thony**
Fixes (partially) #7073 
@lloyddewit @shadrackkibet this just update the database by renaming the menu and dialogue text to `View/Delete Labels` for View Label dialogue.
image

### CODE REVIEW

**Comment by lloyddewit** [comment]
@N-thony Thanks for the database update. 
I assume the change to the `translations` table, fixes the specific issue but it has the side effect that 'View Labels...' will always be translated to 'View/Delete Labels...' for all controls . This may be confusing for future developers. 
I suggest, that you reverse your current changes and instead:
- translation table:
    - View/Delete Labels..., en, View/Delete Labels...
    - View/Delete Labels, en, View/Delete Labels
- form_controls
    - for all affected controls, set the id text to 'View/Delete Labels' or 'View/Delete Labels...' as appropriate

Does that make sense?

Note: In your comment, I also changed 'Partially fixes` to 'Fixes (partially)', otherwise the issue will be closed automatically when this PR is merged.

**Comment by N-thony** [comment]
> * View/Delete Labels

@lloyddewit thanks, I made the requested changes.

**Review by lloyddewit — CHANGES_REQUESTED** [review]
@n-thony The database changes look good, thanks.
One question: `dlgFromLibrary` files are changed, this will cause merge problems for PR #7244 . Are these changes needed?

**Comment by N-thony** [comment]
> @N-thony The database changes look good, thanks. One question: `dlgFromLibrary` files are changed, this will cause merge problems for PR #7244 . Are these changes needed?

@lloyddewit yes.

File: instat/dlgFromLibrary.vb [hunk 1]
```diff
@@ -326,4 +326,11 @@ Public Class dlgFromLibrary
         End If
     End Sub
 
+    Private Sub ucrInputPackages_ControlValueChanged(ucrChangedControl As ucrCore) Handles ucrInputPackages.ControlValueChanged
```
Comment (lloyddewit) [inline instat/dlgFromLibrary.vb:329]: are these functions needed? is it possible to undo the changes to this file?

**Review by lloyddewit — APPROVED** [review]
@N-thony thanks, just one question, if you can resolve this, then I'll approve

File: instat/dlgFromLibrary.vb (hunk 1 above)
Comment (N-thony) [reply instat/dlgFromLibrary.vb]: @lloyddewit I have removed this, take a look. Thanks.

File: instat/dlgFromLibrary.vb [hunk 2]
```diff
@@ -325,5 +325,4 @@ Public Class dlgFromLibrary
             clsImportFunction.AddParameter("data_tables", clsRFunctionParameter:=clsListFunction)
         End If
     End Sub
-
 End Class
```
Comment (lloyddewit) [inline instat/dlgFromLibrary.vb:328]:
```suggestion

End Class
```

**Review by lloyddewit — APPROVED** [review]

**Comment by N-thony** [comment]
@lloyddewit after the quick discussion with Shadrack, the `Label no preview`  on the `Import dataset dialogue` should remain as it was. I have reverted the change in the database
image

**Review by shadrackkibet — APPROVED** [review]

**Pull request merged**